variables.tf

```diff
@@ -966,20 +966,36 @@ variable "defined_tags" {
     # add more tags in each as desired
     vcn = {}
     oke = {
-      cluster = {}
+      cluster           = {}
       persistent_volume = {}
-      service_lb = {}
-      node_pool = {}
-      node = {}
+      service_lb        = {}
+      node_pool         = {}
+      node              = {}
     }
   }
   description = "Tags to apply to different resources."
   type = object({
-    vcn      = map(any),
-    oke      = map(any)
+    vcn = map(any),
+    oke = map(any)
   })
 }
 
+# Cluster autoscaler
+variable "enable_cluster_autoscaler" {
+  description = "Whether to enable cluster autoscaler"
+  type        = bool
+  default     = false
+}
+
+variable "autoscaler_pools" {
+  description = "Node pool to handle cluster autoscaling. Except during cluster upgrade, use only 1 node pool here."
```

Comment:
```suggestion
  description = "Node pool for the Kubernetes Cluster Autoscaler"
```

Reply:
I think the cluster upgrade exception bit should go in the new cluster upgrade doc created from all that content removed from the first README.